index.js

```diff
@@ -67,7 +68,144 @@ function getDefaultConfig(mainTemplate, templatesPath) {
 
       // Add extra function for finding a security scheme by name
       ramlObj.securitySchemeWithName = function (name) {
-        return ramlObj.securitySchemes.find((scheme) => scheme[name]);
+        for (var index = 0; index < ramlObj.securitySchemes.length; ++index) {
```

Comment:
We can do better then a for loop, at the very least we can use `ramlObj.securitySchemes.forEach`. But since all it does it find something that matches and then returns it, really the `.find` should work. If it isn't, let's debug & fix that.